api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -169,6 +169,10 @@ type DRPlacementControlStatus struct {
 	Conditions         []metav1.Condition      `json:"conditions,omitempty"`
 	ResourceConditions VRGConditions           `json:"resourceConditions,omitempty"`
 	LastUpdateTime     metav1.Time             `json:"lastUpdateTime"`
+
+	// Oldest sync time of all PVCs protected in the async mode
+	//+optional
+	LastSyncTime *metav1.Time `json:"lastSyncTime,omitempty"`
```

Comment:
(note) this can move to `status.ResourceConditions.ResourceMeta` as well, and just reflect the value from the VRG. Still thinking if this depth would be useful for a user, or we need to bubble it upward to `status.lastSyncTime`